lib/services/sf.js

```diff
@@ -1378,11 +1378,14 @@ StateConnected.prototype.request = function (options)
  */
 StateConnected.prototype.destroy = function (options)
 {
-  // send out a master token request to terminate the current connection
-  this.createMasterTokenRequest(
+  var requestID = uuidv4();
+  var requestGUID = uuidv4();
```

Comment:
What is the requestGUID? I didn't see it in JDBC code.

Reply:
It's from the .NET connector: https://github.com/snowflakedb/snowflake-connector-net/blob/cfc916791313c510b55e9b2b78ab6ebec6528f39/Snowflake.Data/Core/SFSession.cs#LL266C36-L266C57

The GUID seems to be .NET specific so that could be why it's not on JDBC